vpr/src/draw/draw.h

```diff
@@ -83,6 +83,13 @@ float get_net_alpha();
 
 ezgl::color get_block_type_color(t_physical_tile_type_ptr type);
 
+void breakpoint_info_window(std::string bpDescription, current_information current_info_d);
+
+struct open_windows {
```

Comment:
Again, I think this could be moved into a debug_breakpoints.h. If open_windows is intended only for the breakpoints code, it should probably be moved inside a breakpoints state class or struct.

Reply:
resolved